### PR TITLE
Jliao toronto file

### LINKED ISSUE #19: jliao18 checklist
# My checklist

- [x] Create branch
- [ ] Work locally
- [ ] Create commit
- [ ] Create pull-request
- [ ] Collaborate
- [ ] Merge!

### PR DESCRIPTION
**Pull request by lmyzhao**
Creating a list of places to eat in Toronto, ON

closes #19

### CODE REVIEW

File: jliao-toronto.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Places to eat
+
+- Thai Island
+- Subway
```
Comment (yliu1970) [inline jliao-toronto.md]: 50% soy in their chicken

Comment (tukaidea) [reply jliao-toronto.md]: I heard there will be a lawsuit

Comment (yliu1970) [reply jliao-toronto.md]: are u sure, 50% soy produces lots of gas ...

**Review by yliu1970 — COMMENTED** [review]
comments

**Pull request merged**